.gitignore

```diff
@@ -33,3 +33,4 @@ firebase.json
 src/purifier/dist
 build-system/tasks/performance/cache
 build-system/tasks/performance/results.json
+.python-version
```

Comment:
I think you want to keep the change local

Reply:
I'm happy to revert this. I found that I needed to downgrade to python 2.7 for something to work (it's been a while so what that was escapes me). I suspect others will encounter this and use pyenv to set a version to do AMP development, so you may want this in the future.

As for the example test, that is a great suggestion. Thanks!

Reply:
good point. cc @rsimha